stdlib/public/Concurrency/Task.swift

```diff
@@ -439,22 +445,29 @@ func taskCreateFlags(
 // ==== Task Creation ----------------------------------------------------------
 @available(SwiftStdlib 5.5, *)
 extension Task where Failure == Never {
-  /// Run given `operation` as asynchronously in its own top-level task.
+  /// Runs the given nonthrowing operation asynchronously
+  /// as part of a new top-level task on behalf of the current actor.
   ///
-  /// The `async` function should be used when creating asynchronous work
+  /// Use this function when creating asynchronous work
   /// that operates on behalf of the synchronous function that calls it.
-  /// Like `Task.detached`, the async function creates a separate, top-level
-  /// task.
+  /// Like `Task.detached(priority:operation:)`,
+  /// this function creates a separate, top-level task.
+  /// Unlike `Task.detached(priority:operation:)`,
+  /// the task created by `Task.init(priority:operation:)`
+  /// inherits the priority and actor context of the caller,
+  /// so the operation is treated more like an asynchronous extension
+  /// to the synchronous operation.
   ///
-  /// Unlike `Task.detached`, the task creating by the `Task` initializer
-  /// inherits the priority and actor context of the caller, so the `operation`
-  /// is treated more like an asynchronous extension to the synchronous
-  /// operation.
+  /// You need to keep a reference to the detached task
```

Comment:
```suggestion
  /// You need to keep a reference to the task
```

this isn't detached, so just "to the task" is okey; or "to the unstructured task"

Reply:
Changed here and on the other init below in 946c72cbc73.